debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/OptInfo.java

```diff
@@ -27,6 +27,9 @@
 import static com.ibm.j9ddr.vm29.structure.J9NonbuilderConstants.J9_ROMCLASS_OPTINFO_ENCLOSING_METHOD;
 import static com.ibm.j9ddr.vm29.structure.J9NonbuilderConstants.J9_ROMCLASS_OPTINFO_GENERIC_SIGNATURE;
 import static com.ibm.j9ddr.vm29.structure.J9NonbuilderConstants.J9_ROMCLASS_OPTINFO_PERMITTEDSUBCLASSES_ATTRIBUTE;
+/*[IF INLINE-TYPES]*/
+import static com.ibm.j9ddr.vm29.structure.J9NonbuilderConstants.J9_ROMCLASS_OPTINFO_INJECTED_INTERFACE_INFO;
```

Comment:
I dont think the preprocessor macros work for DDR code. @JasonFengJ9 can you confirm?

Reply:
You can define the flag unconditionally in j9nonbuilder.h

Reply:
@tajila That's correct, JPP doesn't work for DDR code within `debugtools` folder.